src/checker.py

```diff
@@ -89,8 +94,8 @@ def _get_module_data_from_json(self, json_data):
                 size = source.get('size', -1)
                 checker_data = source.get('x-checker-data')
 
-                ext_data = ExternalData(data_type, name, url, sha256sum, size,
-                                        arches, checker_data)
+                ext_data = ExternalData(data_type, pkg_name, name, url,
```

Comment:
From my previous comment, the `pkg_name` would be part of the `checker_data` here.

Reply:
Done